packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -262,7 +262,10 @@ const CreatePassword = ({
                 validateForm,
             }) => (
                 <form onSubmit={handleSubmit}>
-                    <div className='cfd-password-modal__content dc-modal__container_cfd-password-modal__body cfd-password-modal__create-password-content'>
+                    <div
+                        className='cfd-password-modal__content dc-modal__container_cfd-password-modal__body cfd-password-modal__create-password-content'
+                        data-testid='create-password'
```

Comment:
For Testids, It is required to use snake_case and dt prefix.

Reply:
Will rename it